src/simulation/monsters/low/t-z/WealthyCitizen.ts

```diff
@@ -0,0 +1,12 @@
+import LootTable from "../../../../structures/LootTable";
+import SimpleMonster from "../../../../structures/SimpleMonster";
+
+export default new SimpleMonster({
+	id: 13302,
+	name: "Wealthy Citizen",
+	pickpocketTable: new LootTable()
+		.add("Coins", 85)
+		.add("House Keys", 1, 1 / 17)
+		.add("Clue Scroll (Easy)", 1, 1 / 85),
```

Comment:
I wont make you change it, but in the future, always use the osrs naming convention of `House keys`

Reply:
I think this table needs to be changed though.

The weights should be 79 for coins, and so on.